src/Microsoft.SqlTools.ServiceLayer/Connection/ConnectionProviderOptionsHelper.cs

```diff
@@ -155,6 +155,19 @@ internal static ConnectionProviderOptions BuildConnectionProviderOptions()
                         ]
                     },
                     new ConnectionOption
+                    {
+                        Name = "secureEnclaves",
+                        DisplayName = SR.ConnectionConfigOptions_secureEnclaves_displayName,
+                        Description = SR.ConnectionConfigOptions_secureEnclaves_description,
+                        ValueType = ConnectionOption.ValueTypeCategory,
+                        GroupName = GroupNames.Security,
+                        CategoryValues =
+                        [
+                            new CategoryValue { Name = "Disabled", DisplayName = SR.ConnectionConfigOptions_columnEncryptionSetting_category_Disabled },
```

Comment:
should this be `ConnectionConfigOptions_secureEnclaves_category_Disabled`?

Reply:
They could be, but it's the same set of values.  I'll fix the names to make them more clear.